packages/decoupled-kit-health-check/src/classes/GatsbyWordPressHealthCheck.ts

```diff
@@ -73,9 +73,7 @@ export class GatsbyWordPressHealthCheck extends WordPressHealthCheck {
 			? Object.entries(backendVars).filter(([key]) => key === 'WPGRAPHQL_URL')
 			: Object.entries(backendVars);
 
-		const url = /^https:\/\//.test(endpoint) ? endpoint : `https://${endpoint}`;
-
-		this.endpoint = url;
+		this.endpoint = endpoint;
```

Comment:
I think we still need some logic to determine the protocol here, just need to account for `http` as well:
```suggestion
		const url = /^https?:\/\//.test(endpoint) ? endpoint : `https://${endpoint}`;
```

Reply:
@CobyPear  Ahh I now see why this is necessary considering the `PANTHEON_CMS_ENDPOINT` endpoint in prod. However this line is what is allowing the health check to pass if no protocol is set for the `WPGRAPHQL_URL`. Perhaps a solution would be to only use this logic on the `PANTHEON_CMS_ENDPOINT`.